packages/create-workleap/test/spinner.test.ts

```diff
@@ -22,10 +22,8 @@ describe("Given Spinner module", () => {
   test("When start and stop Then work", () => {
```

Comment:
I wonder if there's actually a real value to add those tests. Spinners are very simple code.